Python/README.md

```diff
@@ -0,0 +1,6 @@
+# Steps to run the code samples-
+
+ 1. Download Python from [official site](https://github.com/mahawiki/BackToBasics-Hacktoberfest/tree/main/Python-basics-in-10-programs).
```

Comment:
Everything is great. Just that this link is incorrect. What did you mean here? Download Python folder? It is not possible to download just one folder from a repository. Please change it to something like 'Download the source code from GitHub' or anything similar you want. If you meant to download `python`, then you need to change the link to `python.org/download-url-here`